src/v3/test/integration/__snapshots__/authenticator-enroll-email.test.tsx.snap

```diff
@@ -140,9 +140,15 @@ exports[`Email authenticator enroll when email magic link = true Tests should re
                     <p
                       class="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph emotion-20"
                       data-se="o-form-explain"
-                      id="enroll-authenticator_Description_enroll-authenticator_Description_We_sent_you_a_verification_email_Click_the_verification_link_in_your_email_to_continue_or_enter_the_code_below_okta_email_aut2m2nnGGzJy8uO80g4_okta_email_aut2m2nnGGzJy8uO80g4_8"
+                      id="enroll-authenticator_Description_enroll-authenticator_Description_We_sent_an_email_to_<span_class=strong_no-translate>testUser@oktacom</span>_Click_the_verification_link_in_your_email_to_continue_or_enter_the_code_below_okta_email_aut2m2nnGGzJy8uO80g4_okta_email_aut2m2nnGGzJy8uO80g4_8"
```

Comment:
whoa, this id value is huge and has html in it 😅 . is this expected?

Reply:
Yea, I didn't anticipate this 😅  I will update the id generation method to strip HTML

Reply:
There was one dependency on this auto generated id. I created a tech debt ticket to remove this and update the dependency so it no longer relies on that 🙏 

https://oktainc.atlassian.net/browse/OKTA-595198